src/icetransport.cpp

```diff
@@ -59,6 +59,10 @@ IceTransport::IceTransport(const Configuration &config, Description::Role role,
 
 	mMainLoopThread = std::thread(g_main_loop_run, mMainLoop.get());
 
+	mStreamId = nice_agent_add_stream(mNiceAgent.get(), 1);
```

Comment:
We need stream to be created before STUN Server config, so I moved this here.
Tests is OK but I am not sure ?
Please check it.